modules/kubernetes/opa-gatekeeper/main.tf

```diff
@@ -133,7 +134,7 @@ resource "helm_release" "gatekeeper_templates" {
   chart      = "gatekeeper-library-templates"
   name       = "gatekeeper-library-templates"
   namespace  = kubernetes_namespace.this.metadata[0].name
-  version    = "v0.7.0"
+  version    = "v0.7.1"
```

Comment:
Note the helm chart update. Not a problem but good to know about

Reply:
Needed as I found a bug in the exclude namespace logic.